documentation/examples/remote_storage/example_write_adapter/server.go

```diff
@@ -49,6 +49,11 @@ func main() {
 				}
 				fmt.Printf("\tExemplar:  %+v %f %d\n", m, e.Value, e.Timestamp)
 			}
+
+			for _, hp := range ts.Histograms {
+				h := remote.HistogramProtoToHistogram(hp)
+				fmt.Printf("\tHistogram:  %s\n", (&h).String())
```

Comment:
(`String()` doesn't need to be explicitly called here but I wanted to highlight the methods existence)